spk-config.yaml

```diff
@@ -1,11 +1,11 @@
 # Azure DevOps Configuration
 azure_devops:
-  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
-  hld_repository: "https://dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
-  manifest_repository: "https://dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
-  infra_repository: "https://dev.azure.com/naros32/bedrock_testing_phase2/_git/spk-testing-private-azdo" # Repository URL that contains your terraform templates to be sed for scaffolding and generating infrastructure deployment templates.
-  org: "bhnook" # Your AzDo Org
-  project: "fabrikam" # Your AzDo project
+  access_token: "" # This is a Personal Access Token with permission to modify and access the HLD, manifest and infra repos. Leave this empty if project is public
```

Comment:
For specifics on which permissions this PAT needs, they are listed in this doc: https://github.com/CatalystCode/spk/blob/master/docs/project-service-management-guide.md#generating-personal-access-token

Do you think we should also include those here?

Reply:
Added the link to the doc.